main.go

```diff
@@ -126,6 +130,56 @@ func main() {
     return c.JSONPretty(http.StatusOK, &responseBody, "  ")
   })
 
+  e.GET("/generate-invoice", func(c echo.Context) error {
```

Comment:
can this be a POST rather than a GET? GETs should be idempotent

Reply:
LNURL uses GET :)

Reply:
Thanks, I will update my README PR